### ISSUE TITLE
using System on Bridge Live breaks 'Ready' attribute.

### ISSUE THREAD

**Issue by michaelcheers** [bridgedotnet/Bridge#1302]
[Live Bridge](http://live.bridge.net/#8073a983d4a2bcee81a0b7aed67e5d4a)

**Comment by geoffreymcgill** [comment]
If you include a `using` statement within a Live Bridge code sample, then including all the necessary imports are left up to the user. Currently **Bridge.Html5** is required for use of the `[Ready]` Attribute. That will be changing for Bridge 1.14. 

I revised your sample to include **Bridge.Html5**.

**Example ([Live](http://live.bridge.net/#87a9adac6b2faf311ce9ee7319360451))**

```csharp
using System;
using Bridge.Html5;

public class App
{
    [Ready]
    public static void Main()
    {
    }
}
```

This is question is related to #1303.

**Comment by geoffreymcgill** [comment]
I'm going to mark this issue as a `question`, and close. The fix to not require **Bridge.Html5** will be included with Bridge 1.14.
